### DISCUSSION TITLE
Patches with v2.14.18 and v3.1.8 fixes

### DISCUSSION THREAD

**Discussion by t-babiak** [COVESA/vsomeip#806] [Q&A] [answered]
Hello,

Our project is still based on vsomeip 2.14.16. Unfortunately, upgrading to version 3 is not possible due to other dependencies.

We are currently facing issues with resuming from suspend-to-RAM and a deadlock of a client sending a message.
Looking at the changelog, both issues were already fixed. First with v2.14.18 and the second with v3.1.8.

Unfortunately, neither version is publicly available and both fixes were released as part of a larger update. First is included in the squash commit going from v2.14.16 to v3.1.7.1 and the second between v3.1.7.1 to v3.1.14.1. Since both commits are quite large and contain many other updates, it is difficult to identify the changes related to these particular fixes and determine exactly how they were resolved.

Therefore, I'm wondering if it would be possible to get the patches with only v2.14.18 and v3.1.8 changes, so that we could try to backport it and see if that solves our issues.

Best regards,
Tomas

**Comment by fcmonteiro** [comment, marked as answer]
Hello,
Here's both patches
2.14.18 -> [de0a1b186ed958814a6051cf782d563bced57054.zip](https://github.com/user-attachments/files/17701816/de0a1b186ed958814a6051cf782d563bced57054.zip)

3.1.8 -> [5eebced1e75a5ed6f71c8e613e78a6ac0eeed63e.zip](https://github.com/user-attachments/files/17701393/5eebced1e75a5ed6f71c8e613e78a6ac0eeed63e.zip)

**Comment by t-babiak** [comment]
Thank you very much.
